src/client.js

```diff
@@ -153,11 +153,29 @@ export default class Client {
    * @param {string}    [options.appVersion]                               App Version
    * @return {Promise}                                                     A promise.
    */
-  static initialize(options) {
+  static initialize() {
```

Comment:
@thomasconner this seems to go back to `init` from `initialize`. I understand that you're trying to revert to synchronous initialization, but this would be a significant breaking change.